src/styles/blocks/day.scss

```diff
@@ -0,0 +1,25 @@
+.day {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  @include squareCell($cell-side);
+  background: #eee;
+  border: 1px solid black;
+  cursor: pointer;
+  transition: transform 0.5s linear, background 0.5s linear;
+
+  &:hover {
+    background: pink;
+    transform: translateY(-20px);
+  }
+
+  @for $i from 1 through 31 {
+    &:nth-child(#{$i})::before {
+      content: "#{$i}";
+      font: {
+        family: "Arial", sans-serif;
+        size: 30px;
+      }
```

Comment:
move font styles to html tag

Reply:
Fixed.

Reply:
should you leave font styles here if you moved them to html tag?